paper/paper.md

```diff
@@ -0,0 +1,449 @@
+the same content taught in the lesson, but with different data or variables
+used.
+We hosted the curriculum notebooks in a public GitHub repository to facilitate
+collaborative development and peer review using pull requests.
+In the initial curriculum drafting phase, developers were assigned lesson and
+practice notebooks to write.
+Once the draft of a lesson was completed, the writer opened a pull request and
+asked for review from a different developer.
+The reviewer then provided feedback and approved the pull request to be merged
+into the main branch after the writer made any requested changes.
+This way, more than one person viewed each notebook before it could be
+incorporated into the public curriculum, which reduced mistakes and ensured
+higher quality content. <!-- TODO: cite U-M Carpentries paper for the development model once it's submitted to bioRxiv?-->
+While teaching from the curriculum at the first Data Science Summer Experience,
+instructors took notes on their experience and made revisions afterward.
+Maintainers continue to monitor the repository and resolve issues as they arise.
+
+Following the onset of the COVID-19 pandemic, we quickly pivoted our club to a
+virtual format.
+In preparation for the 2020 Summer Experience, we switched to a flipped
+classroom style following feedback from our club participants that it was too
+difficult to follow along live coding via Zoom.
+We recorded facilitators teaching the lesson notebooks as if they were live
+coding, then shared them along with a link to the lesson notebooks for students
+to code along with while watching the videos.
+Each video shows the Jupyter notebook alongside the facilitator themselves
+teaching.
+This format allowed students to learn at their own pace, then ask questions and
+practice when we met together virtually. <!-- TODO: should this actually go later in the curriculum section?-->
+
+Our Jupyter notebooks and links to the lesson videos can be accessed on GitHub:
+https://github.com/GWC-DCMB/curriculum-notebooks
+
+# Curriculum
+
+Our curriculum was designed for high school students with no prior coding
+experience who are interested in learning Python programming for data science.
+However, this course material would be useful for anyone interested in learning
+basic programming for data analysis, regardless of age or experience level.
+
+Our curriculum features short lessons to deliver course material in “bite sized”
+chunks, followed by practices to solidify the learners' understanding.
+
+## Learning Objectives
+
+Our learning objectives are:
+
+1. Understand fundamental concepts and best practices in coding.
+2. Apply data analysis to real world data to answer scientific questions.
+3. Create informative summary statistics and data visualizations in Python.
+
+These skills provide a solid foundation for basic data analysis in Python and
+participation in our programming exposes students to the many ways coding and
```

Comment:
It seems kind of weird to use this word in the context when most of the time we're actually talking about coding, but I can't think of a better word. Any suggestions?

Reply:
yean